docs/algob-config.md

```diff
@@ -21,6 +21,35 @@ loggingEnabled?: boolean;
 initialDate?: string;
 ```
 
+## Credentials
+
+You can specify Algod Credentials in network object or you can load it from ENV.
+To load it from ENV:
+
+- Method 1
+
+        process.env.ALGOD_ADDR = "127.0.0.1:8080";
+        process.env.ALGOD_TOKEN = "algod_token";
+        let algodCred = algodCredentialsFromEnv();
+
+- Method 2
+
+        process.env.$ALGORAND_DATA = "/path_to/Algorand_Node";
+        let algodCred = algodCredentialsFromEnv();
+
+Similarly for KMD credentials you can either specify credentials in KMD object or load it from ENV.
+To load it from ENV:
+
+- Method 1
+
+        process.env.KMD_ADDR = "127.0.0.1:8080";
+        process.env.KMD_TOKEN = "kmd_token";
+        let kmdCred = KMDCredentialsFromEnv();
+
+- Method 2
+
+        process.env.$KMD_DATA = "/path_to/KMD_DATA";
+        let kmdCred = KMDCredentialsFromEnv();
```

Comment:
Instead of making examples using `process.env` you should do examples using `shell`, eg:

```
export ALGOD_ADDR="127.0.0.1:8080"
...
algob ....
```

An then show in config that user have to use algodCredentialsFromEnv / KMDCredentialsFromEnv

Reply:
okay, should i include a different script in `sample-project` folder and add a instruction to run the shell script before running project ?

Reply:
or should i auto-run it ? i thought running multiple scripts for `sample-project` is not good for user.

Reply:
No need to include the bash scripts. Just provide a simple example.

What do you mean with _auto-run_?

Reply:
for example `project-dev` folder is created using script. similarly running the script for setting the env variables.

Reply:
did you mean provide example in `algob-config.md` or in `sample-project` ?

Reply:
i will update the example in `algob-config.md`. 
and add a comment `you can add ALGOD_ADDR in env using `export ALGOD_ADDR = '127.0.0.1:8080'` ` in `algob-config.js` (sample-project).
will this be good ?

Reply:
> did you mean provide example in algob-config.md or in sample-project ?

in `algob-config.md`

Reply:
You can also add a short note in the sample project, but I want to limit the repetitions. That's why, in `master` I updated it.

Reply:
understood